src/java.base/share/classes/com/sun/crypto/provider/SunJCE.java

```diff
@@ -302,300 +263,261 @@ void putEntries() {
         attrs.put("SupportedModes", "ECB");
         attrs.put("SupportedPaddings", "NOPADDING");
         attrs.put("SupportedKeyFormats", "RAW");
-        ps("Cipher", "ARCFOUR", "com.sun.crypto.provider.ARCFOURCipher",
-                arcFourAliases, attrs);
+        psA("Cipher", "ARCFOUR",
+                "com.sun.crypto.provider.ARCFOURCipher", attrs);
         ps("Cipher", "AESWrap", "com.sun.crypto.provider.AESWrapCipher$General",
                 null, attrs);
-        ps("Cipher", "AESWrap_128",
+        psA("Cipher", "AESWrap_128",
                 "com.sun.crypto.provider.AESWrapCipher$AES128",
-                createAliasesWithOid(aes128Oid+"5"), attrs);
-        ps("Cipher", "AESWrap_192",
+                attrs);
+        psA("Cipher", "AESWrap_192",
                 "com.sun.crypto.provider.AESWrapCipher$AES192",
-                createAliasesWithOid(aes192Oid+"5"), attrs);
-        ps("Cipher", "AESWrap_256",
+                attrs);
+        psA("Cipher", "AESWrap_256",
                 "com.sun.crypto.provider.AESWrapCipher$AES256",
-                createAliasesWithOid(aes256Oid+"5"), attrs);
+                attrs);
 
         attrs.clear();
         attrs.put("SupportedKeyFormats", "RAW");
         if (useNativeChaCha20Cipher && (NativeCrypto.getVersionIfAvailable() >= NativeCrypto.OPENSSL_VERSION_1_1_0)) {
                 ps("Cipher",  "ChaCha20",
                         "com.sun.crypto.provider.NativeChaCha20Cipher$ChaCha20Only",
                         null, attrs);
-                ps("Cipher",  "ChaCha20-Poly1305",
+                psA("Cipher",  "ChaCha20-Poly1305",
                         "com.sun.crypto.provider.NativeChaCha20Cipher$ChaCha20Poly1305",
-                        chachaPolyAliases, attrs);
+                        attrs);
         } else {
                 ps("Cipher",  "ChaCha20",
                         "com.sun.crypto.provider.ChaCha20Cipher$ChaCha20Only",
                         null, attrs);
-                ps("Cipher",  "ChaCha20-Poly1305",
+                psA("Cipher",  "ChaCha20-Poly1305",
                         "com.sun.crypto.provider.ChaCha20Cipher$ChaCha20Poly1305",
-                        chachaPolyAliases, attrs);
+                        attrs);
```

Comment:
While you're here, please correct the indentation of these lines (it's four spaces too much).

Reply:
Removed